CMakeLists.txt

```diff
@@ -745,6 +748,21 @@ ENDMACRO()
 # everything but the source files)
 FUNCTION(IBAMR_SETUP_TARGET_LIBRARY target_library)
   MESSAGE(STATUS "setting up target ${target_library}")
+
+  # Figure out if we are a 2D or a 3D library:
```

Comment:
Can one configuration build both 2D and 3D? Or does a particular configuration now have its spatial dimension fixed at configure time?

Reply:
The default is to build both but we still compile IBAMR2d, IBAMR3d, IBTK2d, IBTK3d as independent libraries.

Reply:
This particular bit of logic refers to if we should link to the 2D or 3D versions of SAMRAI.